benchmarks/cugraph/standalone/bulk_sampling/cugraph_bulk_sampling.py

```diff
@@ -564,17 +565,30 @@ def benchmark_cugraph_bulk_sampling(
     output_sample_path = os.path.join(output_subdir, "samples")
     os.makedirs(output_sample_path)
 
-    batches_per_partition = 200_000 // batch_size
+    if sampling_target_framework == "cugraph_dgl_csr":
+        sampling_kwargs = {
+            "deduplicate_sources": True,
+            "prior_sources_behavior": "carryover",
+            "renumber": True,
+            "compression": "CSR",
+            "compress_per_hop": True,
+            "use_legacy_names": False,
+            "include_hop_column": False,
+        }
+    else:
+        sampling_kwargs = {}
```

Comment:
@alexbarghi-nv , Let me know if this works for `pyg`, we can probably add `pyg specific` args here.

Reply:
```suggestion
    if sampling_target_framework == "cugraph_dgl_csr":
        sampling_kwargs = {
            "deduplicate_sources": True,
            "prior_sources_behavior": "carryover",
            "renumber": True,
            "compression": "CSR",
            "compress_per_hop": True,
            "use_legacy_names": False,
            "include_hop_column": False,
        }
    else:
        # FIXME: Update these arguments when CSC mode is fixed in cuGraph-PyG (release 24.02)
        sampling_kwargs = {
          "deduplicate_sources": True,
          "prior_sources_behavior": "exclude",
          "renumber": True,
          "compression": "COO",
          "compress_per_hop": False,
          "use_legacy_names": False,
          "include_hop_column": True
        }
```